modules/core/src/main/scala/gem/Semester.scala

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+
+import gem.enum.{ Half, Site }
+import java.time._
+import java.time.Month._
+
+/** A (Year, Half) pair. */
+final case class Semester(year: Year, half: Half) {
+
+  /** This Semester plus the given number of years. */
+  def plusYears(n: Int): Semester =
+    copy(year = year.plusYears(n.toLong))
+
+  /** This Semester plus the given number of half-years. */
+  def plusSemesters(n: Int): Semester = {
+    val yy = year.getValue
+    val hs = yy * 2 + half.toInt * yy.signum + n
+    Semester(Year.of(hs / 2), Half.unsafeFromInt(hs % 2))
+  }
+
+  /** The semester immediately following this one. */
+  def next: Semester =
+    plusSemesters(1)
+
+  /** The semester immediately preceding this one. */
+  def prev: Semester =
+    plusSemesters(-1)
+
+  /** Module of various representations of the semester start. */
+  object start {
+    lazy val yearMonth: YearMonth = YearMonth.of(year.getValue, half.startMonth)
+    lazy val localDate: LocalDate = LocalDate.of(year.getValue, half.startMonth, 1)
+    lazy val localDateTime: LocalDateTime = LocalDateTime.of(localDate, LocalTime.MIDNIGHT).minusHours(10) // 2pm the day before
```

Comment:
If I read this correctly, the semester A starts january 31st at 2pm. Is that correct?